context.go

```diff
@@ -124,6 +124,13 @@ func (c *Ctx) LoadProject(path string) (*Project, error) {
 //
 // The second returned string indicates which GOPATH value was used.
 func (c *Ctx) SplitAbsoluteProjectRoot(path string) (string, error) {
+	// the path may lie within a symlinked directory, resolve those first
+	path, err := filepath.EvalSymlinks(path)
+
+	if err != nil {
+		return "", fmt.Errorf("failed attempt to resolve symlinks: %s", err)
```

Comment:
Our patterns around error handling aren't exactly the best right now, but let's be consistent - when wrapping an error returned from a called function, please use `errors.Wrapf()`.